spec/report_spec.rb

```diff
@@ -53,7 +53,8 @@
 
         it 'returns exception' do
           VCR.use_cassette('property_to_create_report_does_not_exist_on_is24') do
-            expect { described_class.retrieve(is24_id, Date.today, Date.today) }.to raise_exception(Immobilienscout::Errors::InvalidRequest)
+            expect { described_class.retrieve(is24_id, Date.today, Date.today) }.to raise_exception(Immobilienscout::Errors::InvalidRequest, 'Resource [realestate] with id '\
+              '[thisIsNotAValidId] not found.')
```

Comment:
Instead of breaking a string across multiple lines with backslashes, I prefer to put one argument per line:

```
expect { described_class.retrieve(is24_id, Date.today, Date.today) }.to raise_exception(
  Immobilienscout::Errors::InvalidRequest,
  'Resource [realestate] with id [thisIsNotAValidId] not found.'
)
```